lib/parsers/aiff_parser.rb

```diff
@@ -18,8 +18,20 @@ class FormatParser::AIFFParser
     'ANNO',
   ]
 
-  def information_from_io(io)
+  NATURES = [:audio].freeze
+  FORMATS = [:aiff].freeze
+
+  def self.natures
+    NATURES
+  end
+
+  def self.formats
+    FORMATS
+  end
+
+  def call(io)
     io = FormatParser::IOConstraint.new(io)
+    io.seek(0)
```

Comment:
These might be redundant because `parse()` already seeks to 0 for every parser in succession. The constrained IO is used in every parser deliberately because parsers usually get tested with just `File` objects and we want to forcibly reduce their API surface.